### ISSUE TITLE
[whistlist] Easy updater

### ISSUE THREAD

**Issue by shakaran** [SimpleMachines/SMF#2349]
Please provide or revamp the current update system for easy one-click updates.

Wordpress has a nice system for upgrades, could you replicate something like that?

**Comment by XinYenFon** [comment]
For feature requests please use our forums: http://www.simplemachines.org/community/index.php?board=3.0

**Comment by illori** [comment]
i really doubt the dev team will revamp the upgrade process at this stage of development.

**Comment by NanoSector** [comment]
SMF has a near one-click system for upgrades, it will prompt you with a link to download the new release and then prompt you to install it in the package manager.

**Comment by illori** [comment]
but that is not between major versions.

**Comment by NanoSector** [comment]
Yeah but major versions aren't released often so not an issue imo. One-click major updates would cause users to look like "WTF HAPPENED TO MY FORUM D:" and complain because usually 1. all their mods break, 2. all their themes break and 3. their forum gets a new look and feel they're not familiar with. So by using the upgrader to switch between major versions you're acknowledging that major things are about to happen because it's not a "simple" upgrade.

TL;DR: Probably not a good idea because of the consequences and confused users, plus major versions take a longer time to release.

**Comment by Oldiesmann** [comment]
A "one-click upgrade" system for major releases isn't going to happen. Too much work and too much of a risk of breaking things.

**Comment by illori** [comment]
@oldiesmann then we should close this?
